gradle-mvn-push.gradle

```diff
@@ -20,9 +20,20 @@ apply plugin: 'signing'
 archivesBaseName = 'android-maps-utils'
 group = 'com.google.maps.android'
 
+task javadoc(type: Javadoc) {
+    failOnError false
```

Comment:
I assume even though `failOnError false`, we'll still get some kind of error message if it fails saying it wasn't successful?

Reply:
Yep - errors and warnings will still be logged.